engines/python/setup/djl_python/rolling_batch/lmi_dist_v2_rolling_batch.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+#
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+# except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+from collections import OrderedDict
+
+from lmi_dist.api import Request
+from lmi_dist.init_engine import engine_from_args
+from vllm import EngineArgs, SamplingParams
+
+from djl_python.rolling_batch.vllm_rolling_batch_base import VllmRollingBatchBase, DTYPE_MAPPER
+from djl_python.properties_manager.vllm_rb_properties import VllmRbProperties
+
+
+class LmiDistRollingBatch(VllmRollingBatchBase):
```

Comment:
I initially used `Mixin` to implement the shared functionality but lot of the code was relying on Base classes' attributes and it would be kind of hard for someone reading just the mixin class and therefore resorted to subclassing later to kind of define an interface. We can always change if this creates problems.

Reply:
what prevent us just changing lmi_dist?